rca/static_src/sass/components/_search-toggle.scss

```diff
@@ -14,7 +14,7 @@
     }
 
     .app__navigation & {
-        margin-top: $grid * 5;
+        margin-top: $grid * 5.75;
```

Comment:
being pinickity on a friday afternoon, apologies ;) `($grid * 5.75)`

Reply:
i'll let you off this time